website/www/site/content/en/blog/beam-2.24.0.md

```diff
@@ -0,0 +1,72 @@
+---
+title:  "Apache Beam 2.24.0"
+date:   2020-07-29 00:00:01 -0800
+# Update date above when committing, as well as date on [download page] link. Check all links for correctness.
+categories:
+  - blog
+authors:
+  - Daniel Oliveira
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+We are happy to present the new 2.24.0 release of Beam. This release includes both improvements and new functionality.
+See the [download page](/get-started/downloads/#2240-2020-07-29) for this release.
+For more information on changes in 2.24.0, check out the
+[detailed release notes](https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12319527&version=12347146).
+
+## Highlights
```

Comment:
@udim - Should we mention FrozenSet changes as a breaking change for 2.24.0 ?

Reply:
Yes:
```
* Python SDK now understands `typing.FrozenSet` type hints, which are not interchangeable with `typing.Set`. You may need to update your pipelines if type checking fails. ([BEAM-10197](https://issues.apache.org/jira/browse/BEAM-10197))
```